scripts/ci-e2e.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+################################################################################
+# usage: ci-e2e.sh
+#  This program runs the e2e tests.
+################################################################################
+
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+cd "${REPO_ROOT}" || exit 1
+
+# shellcheck source=../hack/ensure-go.sh
+source "${REPO_ROOT}/hack/ensure-go.sh"
+# shellcheck source=../hack/ensure-kind.sh
+source "${REPO_ROOT}/hack/ensure-kind.sh"
+# shellcheck source=../hack/ensure-kubectl.sh
+source "${REPO_ROOT}/hack/ensure-kubectl.sh"
+# shellcheck source=../hack/ensure-doctl.sh
+source "${REPO_ROOT}/hack/ensure-doctl.sh"
+
+ARTIFACTS="${ARTIFACTS:-${PWD}/_artifacts}"
+mkdir -p "${ARTIFACTS}/logs/"
+
+SSH_KEY_NAME=capdo-e2e-test
+SSH_KEY_PATH=/tmp/${SSH_KEY_NAME}
+create_ssh_key() {
+    echo "generating new ssh key"
+    ssh-keygen -t rsa -f ${SSH_KEY_PATH} -N '' 2>/dev/null <<< y >/dev/null
+    echo "importing ssh key "
+    doctl compute ssh-key import ${SSH_KEY_NAME} --public-key-file ${SSH_KEY_PATH}.pub --access-token ${DIGITALOCEAN_ACCESS_TOKEN}
+}
+
+remove_ssh_key() {
+    local ssh_fingerprint=$1
+    echo "removing ssh key"
+    doctl compute ssh-key delete ${ssh_fingerprint} --access-token ${DIGITALOCEAN_ACCESS_TOKEN} --force
```

Comment:
`doctl` recognizes the `DIGITALOCEAN_ACCESS_TOKEN` environment variable, so the flag is not needed.